bin/cli-helpers.js

```diff
@@ -13,16 +13,57 @@ function loadVerifyExamplesConfig(configPath) {
   } catch (e) {
     console.log(`Copying initial elm-verify-examples.json to ${configPath}`);
     fsExtra.copySync(
-      path.resolve(__dirname, './templates/elm-verify-examples.json'),
+      path.resolve(__dirname, "./templates/elm-verify-examples.json"),
       configPath
     );
 
-    verifyExamples = require(path.resolve(__dirname, 'templates/elm-verify-examples.json'));
+    verifyExamples = require(path.resolve(
+      __dirname,
+      "templates/elm-verify-examples.json"
+    ));
   }
 
-  return verifyExamples;
+  return resolveTests(configPath, verifyExamples);
 }
 
-module.exports =  {
-    loadVerifyExamplesConfig: loadVerifyExamplesConfig
+function resolveTests(configPath, config) {
+  if (config.tests === "exposed") {
+    /* This is asserting that we want to run for all exposed modules in a package
+     */
+    var elmJson = null;
+    var elmJsonPath = findParentElmJson(path.dirname(configPath));
+    try {
+      elmJson = require(elmJsonPath);
+    } catch (e) {
+      console.error(
+        "Config asks for 'exposed', but could not find elm.json at " +
+          elmJsonPath
+      );
+      process.exit(1);
+    }
+    if (elmJson.type == "package" && config.tests === "exposed") {
```

Comment:
Don't we already know here that `config.tests === "exposed"` is true here?

Reply:
Oh yeah, I was gonna add another feature but changed my mind there.